src/positive.erl

```diff
@@ -2,9 +2,22 @@
 -module(positive).
 -export([is_positive/1, 'really?'/1]).
 
+%% @doc Checks whether an integer is positive or not.
+-spec is_ineeger(any()) -> boolean().
```

Comment:
Right, this is the one I had in mind. We have Travis CI now, so we don't accidentally get into a rut with these.